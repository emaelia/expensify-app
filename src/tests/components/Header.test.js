//react-test-renderer w shallow rendering, checks only given component
import { shallow } from 'enzyme';
import React from 'react';
import Header from '../../components/Header';

test('Should render header correctly', () => {
	const wrapper = shallow(<Header />);
	expect(wrapper).toMatchSnapshot();

	//expect(wrapper.find('h1').text()).toBe('Expensify');
	//const renderer = new reactShallowRenderer();
	//renderer.render(<Header />);
	//Snapshots to track changes of data over time, catches unwanted changes
	//expect(renderer.getRenderOutput()).toMatchSnapshot();
});