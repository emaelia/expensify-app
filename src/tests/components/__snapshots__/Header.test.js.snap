// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render header correctly 1`] = `
<header
  className="header"
>
  <div
    className="content-container"
  >
    <div
      className="header__content"
    >
      <[object Object]
        activeClassName="is-active"
        className="header__title"
        to="/dashboard"
      >
        <h1>
          My Expenses
        </h1>
      </[object Object]>
      <button
        className="button button--link"
        onClick={[Function]}
      >
        Logout
      </button>
    </div>
  </div>
</header>
`;
